packages/workbox-webpack-plugin/src/generate-sw.js

```diff
@@ -80,6 +81,17 @@ class GenerateSW {
    * @private
    */
   async handleEmit(compilation) {
+    // See https://github.com/GoogleChrome/workbox/issues/1790
+    if (this.alreadyCalled) {
```

Comment:
I was thinking about putting this logic in a reusable module, but it a) relies on instance properties of the plugin and b) relies on the `compilation` object being available. It ended up being cleaner just duplicating this logic in both plugin classes, but if there are strong feelings, I could refactor.